CHANGELOG.md

```diff
@@ -6,12 +6,24 @@ All notable changes to this project will be documented in this file. This projec
 
 ### Fixed
 
+- Responses without `Transfer-Encoding: chunked` had no `Content-Length` header set when not done explicitly. This made some clients (e.g. curl) hang.
```

Comment:
This isn't a changelog entry. It's very long. This should be a short title and link to the PR

Same for all the others.

Reply:
our standard is along the lines of this one I pulled as an example from the file:

```
- Update to ponylang/net_ssl 1.3.2 ([PR #69](https://github.com/ponylang/http_server/pull/69))
```